common-crafting.lic

```diff
@@ -88,6 +88,16 @@ module DRCC
     end
   end
 
+  def find_enchanting_room(hometown, override = nil)
+    if override
+      DRCT.walk_to(override)
+    else
+      enchantingrooms = get_data('crafting')['artificing'][hometown]['brazier-rooms']
```

Comment:
💡 I prefer this to be `enchanting_rooms`

Reply:
Okay.